components/link/stories/link.stories.js

```diff
@@ -70,5 +73,88 @@ export default {
 	},
```

Comment:
```suggestion
		docs: {
			story: {
				height: "30px",
			}
		},
	},
```

Reply:
> Miller columns shows an empty message for the ArgTypes doc block

Right, I didn't know what to do about that. I'll check out that PR you left, and experiment a little. Like you said, we can make a card for that work.

I fixed that story height [here](https://github.com/adobe/spectrum-css/pull/2884/commits/93cad164799dbfb58a769c8500718c6f91260a1f)